Cpp_test/lib/ipc-queue-receive.h

```diff
@@ -0,0 +1,30 @@
+#ifndef IPC_QUEUE_RECEIVE_H_
+#define IPC_QUEUE_RECEIVE_H_
+
+#include <climits>
+#include <utility>
+#include <mqueue.h>
+#include <time.h>
+#include <vector>
+#include "ipc.h"
+#include "file-handler.h"
+
+
+class ipcQueueReceive : public IPC
+{
+	protected:
+		mqd_t mqd{};
+		mq_attr attr{0, 10, 8192, 0};
+		unsigned priority{MQ_PRIO_MAX - 1};
+		FileHandler file_handler;
```

Comment:
Now that I think of it, couldn't this be defined in the class IPC already? Since both ipcQueueReceive and ipcQueueSend both have this, you could move this to the base class.